android/scripts/make_rust.sh

```diff
@@ -1,52 +1,91 @@
 #!/bin/bash
 
 # Run from repo's root folder (relative to Gradle build file. TODO do this in Gradle)
-cd ../..
-echo "Running make_rust.sh from $(pwd)"
+
+if [ -d "android/core/gradle" ]; then
+    echo "Starting..."
+else
+    echo "Error: this script must be run in project root"
+    exit 5
+fi
+
+root=$(pwd)
+
+echo "Running make_rust.sh from $root"
 
 # Building  ###########################################################
 
 # See https://github.com/Co-Epi/app-backend-rust/wiki/Building-library-for-Android
 
+if test -z "$PATH_TO_ANDROID_REPO"; then
+    echo "Environment variable PATH_TO_ANDROID_REPO not set. Please set the variable using: 'export PATH_TO_ANDROID_REPO=<your_path_here>'"
+    exit 6
+else
+    echo "\$PATH_TO_ANDROID_REPO is $PATH_TO_ANDROID_REPO"
+    if [ -d "$PATH_TO_ANDROID_REPO" ]; then
+        echo "Will copy files to ${PATH_TO_ANDROID_REPO}..."
+    else
+        echo "Error: ${PATH_TO_ANDROID_REPO} not found. Make sure that you have checked out the corresponding Android project and that the path specified in PATH_TO_ANDROID_REPO environment variable is correct"
+        exit 7
+    fi
+fi
```

Comment:
This doesn't work here, because this script has to run also in the CI, which doesn't have the app clone.

Reply:
@i-schuetz Isnt this script meant for local development and preparation of the release? CI should use a prepackaged (zipped) release.

Reply:
Rust is built (i.e. this script [executed](https://github.com/Co-Epi/app-backend-rust/blob/master/android/core/core/build.gradle#L32)) together with the Android core app (locally and in the CI). 

See https://github.com/Co-Epi/app-android/wiki/Architecture and https://github.com/Co-Epi/app-backend-rust/wiki/Building-library-for-Android

Reply:
Basically, on Android we don't have the flow with copying the binaries locally. It not so necessary, since we can work locally with the Android core library and do all the mapping and testing there.

Reply:
Clear. 
I have updated the wiki page: [Diff for Building library for Android](https://github.com/Co-Epi/app-backend-rust/wiki/Building-library-for-Android/_compare/7ca42f9aa334371c43e938846cd7985db7c75eb9...4e7529fc1c1a106508aa981f0be1c1339d70e020)